torch/testing/_internal/common_cuda.py

```diff
@@ -32,3 +34,52 @@ def initialize_cuda_context_rng():
         for i in range(torch.cuda.device_count()):
             torch.randn(1, device="cuda:{}".format(i))
         __cuda_ctx_rng_initialized = True
+
+
+def tf32_is_not_fp32():
```

Comment:
Better add a comment for this function, too

Reply:
added